pkg/csi/snapshot_restore.go

```diff
@@ -24,6 +24,11 @@ const (
 	createdByLabel          = "created-by-kubestr-csi"
 	clonePrefix             = "kubestr-clone-"
 	snapshotPrefix          = "kubestr-snapshot-"
+
+	basicK8sObjectWait = 1 * time.Second // Sleeps to avoid burning resources on kubectl calls and to make testing more reliable
+
+	PVCKind = "PersistentVolumeClaim"
```

Comment:
If these are being used elsewhere, define them there and not here.

Reply:
I moved these 2. Some of the existing const's are also only used in `csi_ops` but I just left them here